hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2993,6 +2993,21 @@
       FILE_SYSTEM_OPTIMIZED: This layout allows the bucket to support atomic rename/delete operations and
       also allows interoperability between S3 and FS APIs. Keys written via S3 API with a "/" delimiter
       will create intermediate directories.
+      For testing purposes LEGACY layout is also allowed. LEGACY layout is not recommended and will be removed in the
+      future.
+    </description>
+  </property>
+
+  <property>
+    <name>ozone.client.test.ofs.default.bucket.layout</name>
+    <value>FILE_SYSTEM_OPTIMIZED</value>
+    <tag>OZONE, MANAGEMENT</tag>
+    <description>
+      Default bucket layout used by Ozone Manager when buckets are created using Ozone File System.
```

Comment:
Please change the description to convey that this is client side config and for testing purpose only.
How about something like,

"`This configuration is used for testing purpose. Default bucket layout value when buckets are created using ofs. Supported values are LEGACY and FILE_SYSTEM_OPTIMIZED`"